plugin-server/src/main/ingestion-queues/analytics-events-ingestion-consumer.ts

```diff
@@ -18,6 +28,11 @@ export const startAnalyticsEventsIngestionConsumer = async ({
         Consumes analytics events from the Kafka topic `events_plugin_ingestion`
         and processes them for ingestion into ClickHouse.
 
+        Before processing, if an event has overflowed the capacity for its
```

Comment:
Before processing, if isIngestionOverflowEnabled and an event has overflowed the capacity for its